lib/iris/__init__.py

```diff
@@ -187,9 +187,15 @@ def __init__(self, cell_datetime_objects=True, netcdf_promote=True,
         unlimited.  The current default is that the leading dimension is
         unlimited unless otherwise specified.
 
-        The option `clip_latitudes` controls whether the
-        :meth:`iris.coords.Coord.guess_bounds()` method limits the
-        guessed bounds to [-90, 90] for latitudes.
+        .. deprecated:: 2.0.0
+
+            The option `clip_latitudes` is deprecated and will be removed in a
+            future release and the deprecated code paths this option used to
+            toggle have been removed.
+
+            The option `clip_latitudes` controlled whether the
+            :meth:`iris.coords.Coord.guess_bounds()` method would limit the
+            guessed bounds to [-90, 90] for latitudes.
```

Comment:
Given that the functionality is still available, this should probably be in the present tense,

Reply:
However use of the functionality is deprecated and should not be encouraged, so I reckon we should use the past tense, (to be clear: I started the trend elsewhere in this docstring of using the past tense).

Reply:
I noticed your comment elsewhere (with reference to cell datetimes, I think), but I thought in that case the behaviour had actually been removed?